packages/captp/src/captp.js

```diff
@@ -386,7 +475,7 @@ export const makeCapTP = (
    * @returns {RemoteKit<T>}
    * Make a remote promise for `target` (an id in the questions table)
    */
-  const makeRemoteKit = target => {
+  function makeRemoteKit(target) {
```

Comment:
Unless you specifically need both `this`-sensitivity and `new`-constructability, I prefer arrow functions (which forbid having either of those).